lib/config/assets.dart

```diff
@@ -4,6 +4,8 @@ abstract final class Assets {
   static const exportData = 'assets/export-data.png';
   static const planningAhead = 'assets/planning-ahead.png';
   static const whereMoneyGoes = 'assets/where-money-goes.png';
+  static const moneyIn = 'assets/icons/money-in.png';
+  static const moneyOut = 'assets/icons/money-out.png';
 
```

Comment:
Deixar icones separados dos assets de imagens por um espaco

```suggestion
  static const whereMoneyGoes = 'assets/where-money-goes.png';
 
  static const moneyIn = 'assets/icons/money-in.png';
  static const moneyOut = 'assets/icons/money-out.png';
  static const googleIcon = 'assets/icons/google.png';
```